# Q.1
A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units.
A randomly selected sample of cutlets was collected from both units and measured? 
Analyze the data and draw inferences at 5% significance level.
Please state the assumptions and tests that you carried out to check validity of the assumptions.

# Assumptions :
H0 = The diameter of two cutlet is same
H1 = There is significant difference between diameter of two cutlet.
here alpha= 0.05
we used two sample t test for analysing the problem


In [2]:
import pandas as pd 
import numpy as np 
from scipy import stats

In [4]:
df= pd.read_csv('C:/Users/ADMIN/Desktop/Data Science/Assignment_3/Cutlets.csv')
df.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [6]:
p=df['Unit A']

In [7]:
q=df['Unit B']

In [8]:
t_stats,p_value = stats.ttest_ind(p,q)

In [9]:
t_stats,p_value

(0.7228688704678063, 0.4722394724599501)

In [10]:
# For two tail 
alpha = 0.05 
alpha/2

0.025

Here alpha= 0.025 < p_value= 0.472239
Hence, we fail to reject H0 
Inference: There is significant difference in diameter of cut